### DISCUSSION TITLE
운영체제에 따른 월드 구동 차이(?)

### DISCUSSION THREAD

**Discussion by IndiaInk10** [naverz/zepeto-studio-kor#851] [5. Q&A]
IOS 버전은 아무런 문제없이 실행이 잘 되는데, Android버전은 렉이 걸리고 감도도 막 올라가네요...
뭔가 빌드할때 최적화나 세팅 문제인 것 같은데, 혹시 어떤 문제인지 아신다면 알려주시면 정말로 감사하겠습니다!

**Comment by IndiaInk10** [comment]
이유를 알아냈습니다, 라이트 관련 문제였네요. 안드로이드가 조금 더 최적화에 예민하네요.
안드로이드 기준으로 최적화 진행하면 좋은 결과가 있을 것 같습니다.

**Reply by shamKE** [reply]
저도 같은 문제가 발생하네요. 안드로이드 기준 최적화는 어떻게 하나요?? 
아이폰에서만 잘 작동해서 답답하네요. 부탁드립니다ㅠ

**Reply by IndiaInk10** [reply]
@shamKE 일단 저희는 지금 라이트맵 먼저 구웠고요, 그리고 오클루전 컬링 같은 기능들을 사용해서 최적화를 했었습니다. 
https://bit.ly/3zzOLHc 이거 참고해보시면 좋을 것 같습니다.
